apps/site-nft/src/components/App/styles.module.less

```diff
@@ -0,0 +1,24 @@
+@import "~@thecointech/site-semantic-theme/variables.less";
+
+#app {
+  display: grid;
+  grid-template-columns: [full-start] auto [page-start] 173px [content-start] minmax(auto, 837px) [content-end] 173px [page-end] auto [full-end];
+  grid-template-rows: [header-start] 60px [header-end content-start] 1fr [content-end footer-start] 32px [footer-end];
+
+  grid-template-areas:
+    "lgap header header header rgap"
+    "lgap left   main   right  rgap"
+    "lgap footer footer footer rgap";
+}
```

Comment:
This is working really well, it defines the entire layout of the app in one spot. It allows for resizable horizontal & vertical, overflow for the header/footer, but keeps everything consistent with each other.

Note: You can also specify grid guttering, which I think would make the margin issues a lot easier to deal with, but I couldn't figure out how to get the guttering to work nicely with dropshadows